template_library/generics.py

```diff
@@ -0,0 +1,40 @@
+from django.core.exceptions import ValidationError
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+from rest_framework.response import Response
+
+from .serializers import OrganizationCreateUpdateSerializer, OrganizationSerializer
+
+
+class BaseOrganizationListCreateAPIView(ListCreateAPIView):
+    serializer_class = OrganizationSerializer
+
+    def post(self, request, *args, **kwargs):
+        options = {
+            'description': request.POST.get('des', None),
+            'name': request.POST.get('name', None),
+            'slug': request.POST.get('slug', None),
+            'email': request.POST.get('email', None),
+            'url': request.POST.get('org-url', None)
+        }
+        org = self.org_model(**options)
+        try:
+            org.full_clean()
```

Comment:
this is wrong, it needs to be done with a serializer, please study the DRF docs and tutorials to find out how to do it properly